src/components/AskChatGPT/AskChatGPT.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button } from "@mantine/core";
+import { useState } from "react";
+
+interface AskChatGPT {
+    toolName: string;
+    output: string;
+    setChatGPTResponse: (response: string) => void;
+}
+
+const AskChatGPT = ({ toolName, output, setChatGPTResponse }: AskChatGPT) => {
+    const [loading, setLoading] = useState(false);
+
+    /**
+     * Method to send a prompt and additional data to ChatGPT and return the response.
+     * @param prompt The prompt/question for ChatGPT.
+     * @param data Additional context or data to send to ChatGPT (likely the output from a tool).
+     * @returns The response from ChatGPT as a string.
+     */
+    const sendToChatGPT = async (prompt: string, data: string): Promise<string> => {
+        const apiKey = import.meta.env.VITE_OPENAI_API_KEY;
```

Comment:
After adding the API key to the `.env` file, attempting to use ChatGPT on the Traceroute tool failed due to the API key not being set.

This line seems to be looking for a variable named `VITE_OPEN_API_KEY` within the `.env` file. I updated my `.env` file to use this variable name instead and had no issues. I suspect this line can be edited to be:

```
const apiKey = import.meta.env.OPEN_API_KEY;
```

Reply:
I've updated the .env.example and ChatGPT documentation to include the VITE_ prefix. I believe it stops the client seeing the API key in a client-server context (although I'm no expert). While it doesn't make a difference from our development point of view, I figured it is technically more secure?